pkg/tuned/controller.go

```diff
@@ -1088,19 +1090,32 @@ func (c *Controller) changeSyncerTuneD(change Change) (synced bool, err error) {
 	return err == nil, err
 }
 
-func (c *Controller) handleDaemonReloadRestartRequest(change Change, reload, restart bool) error {
+func treatAsImmediate(change Change, inplaceUpdate bool) (bool, string) {
+	if change.nodeRestart {
+		return true, "node restart"
+	}
+	if util.IsImmediateUpdate(change.deferredMode) {
+		return true, "immediate update"
+	}
+	if !inplaceUpdate && change.deferredMode == util.DeferUpdate {
```

Comment:
just to double check -  does `change.deferredMode` needs to be actively cleared ? for instance when its  `change.deferredMode == util.DeferUpdate`

Reply:
No, because `change` is an object which will be automatically destroyed onche the related k8s object has been processed.